test/integration/policy_kyverno_generators_test.go

```diff
@@ -34,7 +34,7 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the kyverno generator policie
 	const kyvernoInstallURL = "https://raw.githubusercontent.com/stolostron/policy-collection/main/community/CM-Configuration-Management/policy-install-kyverno.yaml"
 	const kyvernoInstallPolicy = "policy-install-kyverno"
 
-	Describe("Test stable kyverno generator policies", func() {
+	Describe("Test stable kyverno generator policies", Ordered, func() {
```

Comment:
I think this `Describe` can be removed and the `Ordered` added to its parent. I'm not sure a nested Describe is necessary here.